main.py

```diff
@@ -268,6 +271,32 @@ def main():
 	app.setPalette(palette)
 
 	widget = MainWindow()
+
+	def findXQEMUWindowX11(window, indent):
+			children = window.query_tree().children
+			for w in children:
+				#owner = window.get_full_property(display.get_atom('_NET_WM_PID'), X.AnyPropertyType)
```

Comment:
Couldn't make this work, but ideally we want to find the window by checking the process ID (or by extending QMP to return the window to us)